backend/app/models/record/record.py

```diff
@@ -0,0 +1,31 @@
+from pydantic import BaseModel
```

Comment:
Pydantic convention is to have your own app-specific `BaseModel` that inherits from `pydantic.BaseModel` with the settings you want e.g., extras.